src/x86_64-fma/fft16x16.py

```diff
@@ -509,6 +514,8 @@ def inverse_vfft(reg_t0, reg_t8, reg_t_stride, data_in, reg_row_start=None, reg_
                 elif reg_row_end:
                     CMP(reg_row_end, row_lo)
                     JBE(store_data.end)
+                if relu:
+                    VBLENDVPS(ymm_data_lo, ymm_data_lo, ymm_zero, ymm_data_lo)
```

Comment:
It is preferable to use `VMAXPS(ymm_data_lo, ymm_data_lo, ymm_zero)` for performance reasons (`VBLENDPS` may generate multiple microoperations)